src/App.tsx

```diff
@@ -1,16 +1,29 @@
 import './App.scss';
+import { useState } from 'react';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const addMovie = (movie: any) => {
+    setMovies(
+      [...movies,
+        movie,
+      ],
+    );
+  };
+
   return (
     <div className="page">
       <div className="page-content">
-        <MoviesList movies={moviesFromServer} />
+        <MoviesList movies={movies} />
       </div>
       <div className="sidebar">
-        <NewMovie /* onAdd={(movie) => {}} */ />
+        <NewMovie
+          onAdd={addMovie}
+        />
```

Comment:
You can write it in one line if it's not too long
```suggestion
        <NewMovie onAdd={addMovie} />
```